extensions/bootstrap/ButtonDropdown.php

```diff
@@ -79,11 +79,13 @@ public function run()
         $options = $this->containerOptions;
         $tag = ArrayHelper::remove($options, 'tag', 'div');
 
-        echo Html::beginTag($tag, $this->containerOptions);
-        echo "\n" . $this->renderButton();
-        echo "\n" . $this->renderDropdown();
-        echo "\n" . Html::endTag($tag);
         $this->registerPlugin('button');
+        return implode("\n", [
```

Comment:
wouldn't it be faster and less memory demanding to remove it completely?

specially since the view, layouts and some widgets are captured in a `ob_start()`